versioned_docs/version-2.0.0/quickstart/node-express-mysql.md

```diff
@@ -0,0 +1,293 @@
+---
+id: samples-mysql
+title: NodeJS Sample Application
+sidebar_label: NodeJS - Express + mysql
+description: The following sample app showcases how to use NodeJS framework and the Keploy Platform.
+tags:
+  - javascript
+  - quickstart
+  - samples
+  - examples
+  - tutorial
+keyword:
+  - NodeJS Framework
+  - MySQL
+  - NodeJS
+  - API Test generator
+  - Auto Testcase generation
+---
+
+## Introduction
+
+A simple sample CRUD application and see how seamlessly Keploy integrates with [Express](https://expressjs.com/) and [MySQL](https://www.mysql.com/). Buckle up, it's gonna be a fun ride! 🎢
+
+## Pre-Requisite 🛠️
+
+- Install WSL (`wsl --install`) for <img src="/docs/img/os/windows.png" alt="Windows" width="3%" /> Windows.
+- Install Colima( `brew install colima && colima start` ) for <img src="/docs/img/os/macos.png" alt="MacOS" width="3%" /> MacOs.
+
+## Get Started! 🎬
+
+## Setup application
+
+Clone the repository and move to js-mysql folder
+
+```bash
+git clone https://github.com/keploy/samples-typescript && cd samples-typescript/js-mysql
+
+# Install the dependencies
+npm install
+```
+
+## Installation 📥
+
+Depending on your OS, choose your adventure:
+
+- <details>
+     <summary><img src="/docs/img/os/linux.png" alt="Linux" width="3%" /> Linux or <img src="/docs/img/os/windows.png" alt="Windows" width="3%" /> Windows</summary>
+
+  Alright, let's equip ourselves with the **latest Keploy binary**:
+
+  ```bash
+  curl --silent --location "https://github.com/keploy/keploy/releases/latest/download/keploy_linux_amd64.tar.gz" | tar xz -C /tmp
+
+  sudo mkdir -p /usr/local/bin && sudo mv /tmp/keploy /usr/local/bin && keploy
+  ```
+
+  If everything goes right, your screen should look a bit like this:
+
+     <img src="/docs/img/code-snippets/install-keploy-logs.png" alt="Test Case Generator" width="50%" />
+
+  Moving on...
+     <details>
+     <summary style={{ fontWeight: 'bold', fontSize: '1.17em', marginLeft: '0.5em' }}> Run App with <img src="/docs/img/os/docker.png" alt="Docker Container" width="3%" /> Docker </summary>
+
+  #### Add alias for Keploy:
+
+  ```bash
+  alias keploy='sudo docker run --pull always --name keploy-v2 -p 16789:16789 --privileged --pid=host -it -v "$(pwd)":/files -v /sys/fs/cgroup:/sys/fs/cgroup -v /sys/kernel/debug:/sys/kernel/debug -v /sys/fs/bpf:/sys/fs/bpf -v /var/run/docker.sock:/var/run/docker.sock -v '"$HOME"'/.keploy-config:/root/.keploy-config -v '"$HOME"'/.keploy:/root/.keploy --rm ghcr.io/keploy/keploy'
+  ```
+
+  ### Lights, Camera, Record! 🎥
+
+  Fire up the application and MySQL instance with Keploy. Keep an eye on the two key flags:
+  `-c`: Command to run the app (e.g., `docker compose up`).
+
+  `--containerName`: The container name in the `docker-compose.yml` for traffic interception.
+
+  ```sh
+  keploy record -c "docker compose up" --containerName "nodesqlapp"
```

Comment:
https://github.com/Sonichigo/samples-typescript/blob/main/js-mysql/docker-compose.yaml

In the docker-compose.yml file the containerName is not present. there this method won't work unless add manually by user following the guide.